build_files/org_chromium_libunwind.BUILD

```diff
@@ -1,27 +0,0 @@
-package(default_visibility = ['//visibility:public'])
```

Comment:
Hmm, how is `libunwind` broken here?

Reply:
@vladlosev, it's no longer used to build Chromium for another other than fuchsia: https://bugs.chromium.org/p/chromium/issues/detail?id=803679

Reply:
IIRC, `libunwind` is used by exception handling. Chrome, like most Google code, does not use exceptions, hence no need for `libunwind`. Is it possible some of our code (or dependencies) will need them?

Reply:
> Is it possible some of our code (or dependencies) will need them?

I think it could be possible in the future, but since this repo is, at present, concerned with building a Bazel toolchain based strictly upon Chromium's LLVM toolchain, I think that we can follow conventions there until we have a good reason to break away. 

Since we are not using libunwind presently in any consumers of this repo, I think it would be better to remove the broken code and then re-add a working version later (or via contributions) when necessary.